modules/ROOT/pages/managing-instances/instance-actions.adoc

```diff
@@ -187,3 +187,16 @@ Type the exact name of the instance (as instructed) to confirm your decision, an
 ====
 There is no way to recover data from a deleted Aura instance.
 ====
+
+== Mark an instance as production
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+label:AuraDB-Business-Critical[]
+
+Admins can mark a database instance as `production` in AuraDB Virtual Dedicated Cloud and AuraDB Business Critical.
+
+When Neo4j updates Aura Database Versions, instances marked as `production` are last to have the Aura Database Version updated. Because updates are applied to these database instances after other database instances recieve updates, you can monitor any potential impact on less critical databases first.
```

Comment:
```suggestion
When Neo4j updates Aura Database versions, instances marked as `production` are last to have the Aura Database version updated. 
Because updates are applied to these database instances after other database instances receive updates, you can monitor any potential impact on less critical instances first.
```